modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/buffer/BufferedIndexerInvocationHandler.java

```diff
@@ -113,11 +113,11 @@ public Object invoke(Object proxy, Method method, Object[] args)
 
 			Long classPK = (Long)classedModel.getPrimaryKeyObj();
 
-			if (args[0] instanceof ResourcedModel) {
-				ResourcedModel resourcedModel = (ResourcedModel)args[0];
-
-				classPK = resourcedModel.getResourcePrimKey();
-			}
+//			if (args[0] instanceof ResourcedModel) {
```

Comment:
Hi @brandizzi, the only way i found to make the `WikiPageModelDocumentContributor` works was by removing this validation, but i am not sure of the impacts it may cause. The tests are still passing.